### PR TITLE
Fixed some Typos

### PR DESCRIPTION
**Pull request by Levin779**
Fixed some minor Typos.

### CODE REVIEW

**Review by harish-sethuraman — APPROVED** [review]
LGTM

File: content/docs/conditional-rendering.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ It works because in JavaScript, `true && expression` always evaluates to `expres
 
 Therefore, if the condition is `true`, the element right after `&&` will appear in the output. If it is `false`, React will ignore and skip it.
 
-Note that returning a falsy expression will still cause the element after `&&` to be skipped but will return the falsy expression. In the example below, `<div>0</div>` will be returned by the render method.
```
Comment (gaearon) [inline content/docs/conditional-rendering.md:155]: "falsy" and "false" are two different terms.

File: content/tutorial/tutorial.md [hunk 2]
```diff
@@ -1045,7 +1045,7 @@ Let's `map` over the `history` in the Game's `render` method:
 
 **[View the full code at this point](https://codepen.io/gaearon/pen/EmmGEa?editors=0010)**
 
-As we iterate through `history` array, `step` variable refers to the current `history` element value, and `move` refers to the current `history` element index. We are only interested in `move` here, hence `step` is not getting assigned to anything.
```
Comment (gaearon) [inline content/tutorial/tutorial.md:1048]: "to `move`" doesn't make sense in this context

**Comment by gaearon** [comment]
Thanks for the PR, but half of these are completely subjective and the other half is wrong and makes the text incorrect.

**Pull request closed without merging**